src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -538,6 +584,52 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     });
   }
 
+  private _handleSeekBackward() {
+    this._seekDelta -= SEEK_BUTTON_INCREMENT;
+    this._debouncedSeek();
+  }
+
+  private _handleSeekForward() {
+    this._seekDelta += SEEK_BUTTON_INCREMENT;
+    this._debouncedSeek();
+  }
+
+  private _debouncedSeek() {
+    if (this._seekTimeout) {
+      clearTimeout(this._seekTimeout);
+    }
+    this._seekTimeout = window.setTimeout(
+      () => this._executeSeek(),
+      SEEK_BUTTON_DEBOUNCE
+    );
+  }
+
+  private _executeSeek() {
+    const currentPosition = getCurrentProgress(this._stateObj!);
+    let newPosition = currentPosition + this._seekDelta;
+    newPosition = Math.min(
+      newPosition,
+      this._stateObj!.attributes.media_duration!
+    );
+    newPosition = Math.max(newPosition, 0);
+
+    this.hass!.callService("media_player", "media_seek", {
+      entity_id: this._config!.entity,
+      seek_position: newPosition,
+    });
+    this._seekDelta = 0;
+    this._seekTimeout = undefined;
+  }
+
+  private _formatSeekDelta() {
```

Comment:
Could be converted into a getter or extracted into a function outside the class. Not sure what's preferred.